src/commons/sideContent/SideContentHtmlDisplay.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+
+export interface SideContentHtmlDisplayProps {
```

Comment:
Not sure why this particular coding standard was set, but a `type` should be used here instead of an `interface`. See https://github.com/source-academy/frontend/wiki/Coding-Standard#interface-and-types.